src/store/modules/picklist/actions.ts

```diff
@@ -0,0 +1,37 @@
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import PicklistState from './PicklistState'
+import * as types from './mutation-types'
+import { PicklistService } from '@/services/PicklistService'
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+
+const actions: ActionTree<PicklistState, RootState> = {
+  async createPicklist ({ commit }, payload) {
```

Comment:
We do not need state for this, we could use methods. We are not storing anything shared in state